cspell.json

```diff
@@ -24,6 +26,8 @@
 		"packagejson",
 		"quickstart",
 		"ruleset",
+		"substringvalue",
+		"substringvalueeeeee",
```

Comment:
I can remove the other non-words from cspell.json, but I have to add "altano" to the dictionary because it's spell checking package names in package.json

Reply:
ye ole #381 -> https://github.com/streetsidesoftware/cspell/issues/3215